.travis.yml

```diff
@@ -2,14 +2,14 @@ services:
   - docker
 
 before_install:
+  - mkdir dest
+  - make librdkafka LIBRDKAFKA_CONF=--prefix=dest
+  - export LD_LIBRARY_PATH=LD_LIBRARY_PATH:dest/lib
```

Comment:
I pulled this locally and am testing. what is the purpose of the `dest` directory? I'm finding it empty after I run `make librdkafka`

Reply:
i got this to populate by changing it to `--prefix=../dest`, will see if that makes any difference